### PR TITLE
Update campaigns

### PR DESCRIPTION
**Pull request by scarletnorberg**
Adding in RunIIpp5Spring18GSFixBeamspot this new campaign no pilot PRCAMPAIGNS-154

Removing FNAL from the New Campaigns: RunIISummer20UL17DIGIPremix in order to force MSTransfer with a dummy WF to transfer to CERN so we will have the PU input at both FNAL and CERN. PRCAMPAIGNS-140

### CODE REVIEW

**Comment by haozturk** [comment]
@scarletnorberg why is `RunIIAutumn18NanoAOD` removed? Is this a known change?

**Comment by scarletnorberg** [comment]
If we moved back in it was because of work that needed to get through that had fallen through the cracks in general that should not be included as we are on the nanoAODv7 version for the RunIIAutumn18/RunIIFall17/RunIISummer16 campaigns.

File: campaigns.json [hunk 1]
```diff
@@ -781,13 +775,13 @@
...
+      "T2_US_Caltech"
+    ], 
     "force-complete": 0.99, 
     "fractionpass": 0.95, 
     "go": true, 
-    "lumisize": -1,
-    "SiteBlacklist": [
-      "T2_US_Caltech"
-    ],     
+    "lumisize": -1,
```
Comment (scarletnorberg) [inline campaigns.json:784]: This seems to have moved

File: campaigns.json [hunk 2]
```diff
@@ -835,12 +829,12 @@
...
   "RunIIFall18GS": {
-    "fractionpass": 0.95, 
-    "go": true, 
-    "lumisize": -1,
     "SiteBlacklist": [
       "T2_US_Caltech"
     ], 
+    "fractionpass": 0.95, 
+    "go": true, 
+    "lumisize": -1,
```
Comment (scarletnorberg) [inline campaigns.json:837]: Just moved

File: campaigns.json [hunk 3]
```diff
@@ -858,13 +852,13 @@
     "tune": true
   }, 
   "RunIISummer15GS": {
+    "SiteBlacklist": [
+      "T2_US_Caltech"
+    ],
```
Comment (scarletnorberg) [inline campaigns.json:857]: Just mvoed

File: campaigns.json [hunk 4]
```diff
@@ -1122,8 +1116,8 @@
         "pending": 0
       }
     }, 
-    "resize": "auto",
     "partial_copy": 0.95, 
+    "resize": "auto",
```
Comment (scarletnorberg) [inline campaigns.json:1120]: just mvoed

File: campaigns.json [hunk 5]
```diff
@@ -1747,20 +1741,20 @@
...
-    "partial_copy": 0.95,
+    "maxcopies": 1, 
     "overflow": {
       "PU": {
         "force": false, 
         "max": 0, 
         "pending": 0
       }
     }, 
+    "partial_copy": 0.95,
```
Comment (scarletnorberg) [inline campaigns.json:1752]: Just moved

File: campaigns.json [hunk 6]
```diff
@@ -2061,7 +2055,6 @@
   }, 
   "RunIISummer20UL17DIGIPremix": {
     "SecondaryLocation": [
-      "T1_US_FNAL", 
       "T2_CH_CERN"
```
Comment (scarletnorberg) [inline campaigns.json:2058]: Needed to be removed

File: campaigns.json [hunk 7]
```diff
@@ -2382,24 +2374,25 @@
         "pending": 0
       }
     }, 
-    "resize": "auto",
+    "partial_copy": 0.95,
```
Comment (scarletnorberg) [inline campaigns.json:2377]: Just mvoed

File: campaigns.json [hunk 8]
```diff
@@ -2537,8 +2530,7 @@
     "secondaries": {
       "/MinBias_TuneCUETP8M1_2017_5p02TeV-pythia8/RunIIpp5Spring18GS-94X_mc2017_realistic_v10For2017G_v3-v2/GEN-SIM": {
         "SiteWhitelist": [
-          "T2_US_MIT", 
-          "T2_DE_DESY"
+          "T2_US_MIT"
```
Comment (scarletnorberg) [inline campaigns.json:2533]: Checked and when looking at RUCIO this was only at MIT and not at DESY so removed DESY

File: campaigns.json [hunk 9]
```diff
@@ -2569,7 +2561,27 @@
...
+        "T2_US_Caltech", 
+        "T2_US_Florida", 
+        "T2_US_Nebraska", 
+        "T2_US_UCSD", 
+        "T2_US_Wisconsin"
+      ]
+    }, 
+    "resize": "auto", 
+    "tune": true
+  },
```
Comment (scarletnorberg) [inline campaigns.json:2584]: Adding this camapign

**Comment by haozturk** [comment]
@scarletnorberg The secondary name should also be updated for `RunIISummer20UL17DIGIPremix`: https://its.cern.ch/jira/browse/PRCAMPAIGNS-140

**Comment by scarletnorberg** [comment]
You are right I just saw UL17 and checked that the file existed but it was the 19UL which is what they tested the WF in for the pilot.

**Comment by scarletnorberg** [comment]
@haozturk Updated the file good catch I just say 17UL and checked that it existed the JIRA is updated also

**Pull request merged**